QModManager/Patching/QModFactory.cs

```diff
@@ -52,7 +52,7 @@ internal List<QMod> BuildModLoadingList(string qmodsDirectory)
                 if (status != ModStatus.Success)
                 {
                     Logger.Debug($"Mod '{mod.Id}' will not be loaded");
-                    earlyErrors.Add(mod);
+                    if (status != ModStatus.Merged) earlyErrors.Add(mod);
```

Comment:
We should inform this somewhere.
```CS
 if (status != ModStatus.Merged)
    earlyErrors.Add(mod);
else
    Logger.Info($"'{mod.Id}' has been merged into QModManager. The DLL in the QMods folder will not be loaded and can be deleted.");
```

Reply:
It's in `SummaryLogger`. Although it currently doesn't get logged (check discord)